Src/CSharpier/CSharpier.csproj

```diff
@@ -2,7 +2,7 @@
     <Import Project="../../CSharpier.Build.props" />
     <PropertyGroup>
         <PackageId>CSharpier.Core</PackageId>
-        <TargetFrameworks>net5.0;net6.0</TargetFrameworks>
+        <TargetFrameworks>net5.0;net6.0;netstandard2.0</TargetFrameworks>
```

Comment:
Is there any particular reason this has to be `netstandard2.0`?
IIRC `netstandard2.1` includes the `Index` type and you won't have to do this shim.